app/javascript/schools/courses/components/evaluation_criteria_editor/EvaluationCriterionEditor__Form.re

```diff
@@ -268,7 +268,7 @@ let make = (~evaluationCriterion, ~courseId, ~addOrUpdateCriterionCB) => {
             />
             <School__InputGroupError
               message="Enter a valid name"
-              active={state.dirty && state.name |> String.length < 1}
+              active={state.dirty && state.name |> String.trim |> String.length < 1}
```

Comment:
There's a mismatch here as well, but in reverse. The `saveDisabled` function requires a minimum length of 2 **and** it doesn't strip `name` before checking. `saveDisabled` should be updated to strip and check for length `> 0`,  to match this error message.